panic: runtime error: invalid memory address or nil pointer dereference

### Linux distribution

Zorin 18 (Pro) based on Ubuntu 24.04 LTS

### Installation method

Flatpak

### Vinegar version

63a286cdd602b17885fead4abd85ad5fcf3da8f2ced4b91f922d8236cc11c937

### What problem did you encounter?

Installation:
```bash
$ flatpak run org.vinegarhq.Vinegar
12:42:19 INF Logging to file path=/home/khtsly/.var/app/org.vinegarhq.Vinegar/cache/vinegar/logs/2026-01-06T12:42:19+07:00.log
12:42:32 INF Saving configuration!
12:42:45 INF Saving configuration!
12:42:56 INF Saving configuration!
12:42:59 INF Saving configuration!
12:43:08 INF Fetched Wine build tag=10.20+80f87fd released=2025-12-28T18:45:10.000Z
12:43:08 INF Downloading Wine build url=https://github.com/vinegarhq/kombucha/releases/download/10.20%2B80f87fd/kombucha-10.20%2B80f87fd.tar.xz
12:43:14 INF Updated Wine
12:43:14 INF Set local Wine installation tag=10.20+80f87fd
12:43:19 INF Setting up Wine first-time=true
12:43:19 INF wine: Updating Wine Prefix
12:43:20 WIN wine: failed to start L"C:\\windows\\syswow64\\rundll32.exe": c0000135
12:43:20 WIN 002c:err:setupapi:do_file_copyW Unsupported style(s) 0x10
12:43:20 WIN 00f8:err:setupapi:do_file_copyW Unsupported style(s) 0x10
12:43:20 WIN 00f8:err:setupapi:do_file_copyW Unsupported style(s) 0x10
12:43:22 INF Updating Paths
12:43:35 INF Checking Wineprefix
12:43:35 INF Acquiring user authentication
12:43:35 WRN Retrieving authenticated user failed err="credential manager missing"
12:43:35 INF Checking WebView against=143.0.3650.96
12:43:35 INF Fetching WebView upload=143.0.3650.96
12:43:36 INF Downloading WebView catalog="AcmMtmz3dxtqpux0Y1_kZoRCGvI9RzX_6Nrs8PBALGk="
12:44:04 INF Installing WebView version=143.0.3650.96 path=/home/khtsly/.var/app/org.vinegarhq.Vinegar/cache/vinegar/webview-143.0.3650.96.exe
12:44:29 INF Checking for updates
12:44:30 INF Using Deployment guid=version-34c9847319974d28 channel=LIVE
12:44:30 INF Installing Studio new=version-34c9847319974d28
12:44:30 INF Finding Mirror
12:44:31 INF Fetching Package List
12:44:31 INF Fetching Installation Directives
12:44:33 INF Installing Packages count=35 dir=/home/khtsly/.var/app/org.vinegarhq.Vinegar/data/vinegar/versions/version-34c9847319974d28
12:44:33 INF Downloading package name=RobloxStudio.zip sum=f1fee47586602d419b6f04197c549908
12:44:33 INF Downloading package name=content-models.zip sum=ff7510896b3b211e561900b0c26ee502
12:44:33 INF Downloading package name=content-platform-dictionaries.zip sum=c15bf080c272b587b050fdcf0c920e4e
12:44:33 INF Downloading package name=extracontent-scripts.zip sum=eee8375d7b64c596694c54085dcd8269
12:44:33 INF Downloading package name=content-avatar.zip sum=6a7a121b4097b869bb3cf426c92b0f37
12:44:33 INF Downloading package name=content-textures3.zip sum=10461fdfb35f35cca13dbe35dcf21343
12:44:33 INF Downloading package name=ApplicationConfig.zip sum=9e6ce8a71b433d3e6802b3438b418c2d
12:44:33 INF Downloading package name=redist.zip sum=c9622ecfbec2c14d30f390909c563124
12:44:33 INF Downloading package name=ssl.zip sum=805c69c51a176f39a9b7793d8907133a
12:44:33 INF Downloading package name=content-api-docs.zip sum=ec3c1c1a05528bbaa10f6b8e2759118e
12:44:33 INF Downloading package name=extracontent-models.zip sum=cd0f0ecd784dc5d45297f42b1c2a1952
12:44:33 INF Downloading package name=WebView2RuntimeInstaller.zip sum=f2515c0fc7649176e200cc3cbe0bffb5
12:44:33 INF Downloading package name=extracontent-translations.zip sum=697aaa85b0cd30b5198ce95b1da77fee
12:44:33 INF Downloading package name=content-sky.zip sum=1d0390337d1a4a58e5514be1a9481ad6
12:44:33 INF Downloading package name=Plugins.zip sum=b4b04085aaf099cec525e5368992fbf5
12:44:33 INF Downloading package name=RibbonConfig.zip sum=1eb7a04d6b1ccb75b47bbfef41feab3f
12:44:33 INF Downloading package name=content-qt_translations.zip sum=9ac41564393da36084990e6dd9d08f50
12:44:33 INF Downloading package name=StudioFonts.zip sum=a9ab0fb1b76e33f406e92991b71f3646
12:44:33 INF Downloading package name=extracontent-luapackages.zip sum=f06487ae31a091929e3691bbc7c02cf7
12:44:33 INF Downloading package name=content-platform-fonts.zip sum=8f379ec2b22ff106b837d79f7fdbf0d8
12:44:33 INF Downloading package name=studiocontent-models.zip sum=6b068c7b9041ba046c47239c0028aea6
12:44:33 INF Downloading package name=BuiltInPlugins.zip sum=840fae7a19e90df41938c1915fdbc7b5
12:44:33 INF Downloading package name=extracontent-textures.zip sum=bebd2d622e4acf2b160878ff02350cf3
12:44:33 INF Downloading package name=content-textures2.zip sum=a7d45920c6a193206a6abae040eb50d0
12:44:33 INF Downloading package name=content-configs.zip sum=7c1119ae6435b35d30a63f1051027dbd
12:44:33 INF Downloading package name=Libraries.zip sum=3878459412d843f11aeccad8ebe1d01d
12:44:33 INF Downloading package name=content-sounds.zip sum=eb9dd4d0637fd6ae21747e6b6a281d21
12:44:33 INF Downloading package name=BuiltInStandalonePlugins.zip sum=ffdf1be8a11b61e64ce97170d06156b2
12:44:33 INF Downloading package name=content-fonts.zip sum=d1e80bdacf3d7f5bd1f63eadfe30606a
12:44:33 INF Downloading package name=content-studio_svg_textures.zip sum=e7a11707eb441e031093bb18a117d22c
12:44:33 INF Downloading package name=LibrariesQt5.zip sum=19364dd7196cea304a587e58492c85ec
12:44:33 INF Downloading package name=shaders.zip sum=2438e800d9e942776833e7a858b56132
12:44:33 INF Downloading package name=WebView2.zip sum=210bcdc7860b2ecc15a11371fe0e034b
12:44:33 INF Downloading package name=content-terrain.zip sum=97327884177381d0f42d428110b4c516
12:44:33 INF Downloading package name=studiocontent-textures.zip sum=8bc1c8be1ce3bd938b394efa489510e3
12:44:33 INF Extracting package name=content-avatar.zip dest=content/avatar
12:44:33 INF Extracting package name=RibbonConfig.zip dest=RibbonConfig
12:44:33 INF Extracting package name=content-terrain.zip dest=PlatformContent/pc/terrain
12:44:33 INF Extracting package name=ssl.zip dest=ssl
12:44:33 INF Extracting package name=content-platform-dictionaries.zip dest=PlatformContent/pc/shared_compression_dictionaries
12:44:34 INF Extracting package name=ApplicationConfig.zip dest=ApplicationConfig
12:44:34 INF Extracting package name=content-models.zip dest=content/models
12:44:34 INF Extracting package name=WebView2.zip dest=.
12:44:34 INF Extracting package name=studiocontent-models.zip dest=StudioContent/models
12:44:35 INF Extracting package name=content-configs.zip dest=content/configs
12:44:35 INF Extracting package name=extracontent-translations.zip dest=ExtraContent/translations
12:44:36 INF Extracting package name=content-sounds.zip dest=content/sounds
12:44:37 INF Extracting package name=content-qt_translations.zip dest=content/qt_translations
12:44:38 INF Extracting package name=Plugins.zip dest=Plugins
12:44:39 INF Extracting package name=redist.zip dest=.
12:44:40 INF Extracting package name=WebView2RuntimeInstaller.zip dest=WebView2RuntimeInstaller
12:44:40 INF Extracting package name=extracontent-scripts.zip dest=ExtraContent/scripts
12:44:43 INF Extracting package name=content-textures2.zip dest=content/textures
12:44:43 INF Extracting package name=content-sky.zip dest=content/sky
12:44:44 INF Extracting package name=extracontent-textures.zip dest=ExtraContent/textures
12:44:45 INF Extracting package name=studiocontent-textures.zip dest=StudioContent/textures
12:44:45 INF Extracting package name=content-textures3.zip dest=PlatformContent/pc/textures
12:44:46 INF Extracting package name=content-fonts.zip dest=content/fonts
12:44:47 INF Extracting package name=Libraries.zip dest=.
12:44:49 INF Extracting package name=content-studio_svg_textures.zip dest=content/studio_svg_textures
12:44:50 INF Extracting package name=LibrariesQt5.zip dest=.
12:44:50 INF Extracting package name=content-api-docs.zip dest=content/api_docs
12:44:51 INF Extracting package name=extracontent-models.zip dest=ExtraContent/models
12:44:53 INF Extracting package name=extracontent-luapackages.zip dest=ExtraContent/LuaPackages
12:44:53 INF Extracting package name=shaders.zip dest=shaders
12:44:57 INF Extracting package name=content-platform-fonts.zip dest=PlatformContent/pc/fonts
12:45:14 INF Extracting package name=BuiltInStandalonePlugins.zip dest=BuiltInStandalonePlugins
12:45:19 INF Extracting package name=BuiltInPlugins.zip dest=BuiltInPlugins
12:45:33 INF Extracting package name=RobloxStudio.zip dest=.
12:45:33 INF Extracting package name=StudioFonts.zip dest=StudioFonts
12:45:34 INF Writing AppSettings
12:45:34 INF Successfully installed! guid=version-34c9847319974d28
12:45:34 INF Applying FFlags
12:45:34 INF Updating Wine DPI dpi=96
12:45:34 INF Running Studio! cmd="/home/khtsly/.var/app/org.vinegarhq.Vinegar/data/vinegar/kombucha/bin/wine /home/khtsly/.var/app/org.vinegarhq.Vinegar/data/vinegar/versions/version-34c9847319974d28/RobloxStudioBeta.exe"
12:45:34 INF Registered with GameMode response=0
12:45:34 WIN 01ec:err:winediag:wined3d_dll_init Using the Vulkan renderer.
12:45:35 RBX [FLog::BackendConfigCacheManager] Header version / timestamp is empty
12:45:35 RBX [FLog::BackendConfigManager] startTimestampMs: 1767678335064. Thread start took 0ms
12:45:35 RBX [FLog::Output] settingsUrl: https://clientsettingscdn.roblox.com/v2/settings/application/PCStudioApp
12:45:35 RBX [FLog::UpdateRequest] Didn't receive 'flagOnly' property for channel. Assuming it's false.
12:45:35 RBX [FLog::Output] Settings X-Signature-Ed25519 header not found
12:45:35 RBX [FLog::BackendConfigManager] Wait flag took 184ms. Flag Source remoteCDN
12:45:35 RBX [FLog::BackendConfigManager] Wait guac took 212ms. Guac Source remoteCDN. Total startup config fetching time = 396
12:45:35 RBX [FLog::BackendConfigManager] Wait latest version took 0ms. Total startup config fetching time = 396
12:45:35 RBX [FLog::ClientSettings] LoadIxpSettingsFromLocal path: \"StudioIxpSettings.json\"
12:45:35 RBX [FLog::ClientSettings] LoadIxpSettingsFromLocal: Couldn't fetch any data from local
12:45:35 RBX [FLog::Output] Session GUID is 33E14FF0-FEB5-4DE2-9A14-D5CADB2B4E07
12:45:35 RBX [FLog::Output] Machine GUID is 623D4ADA-06CF-4743-9752-E656F5F42F62
12:45:35 RBX [FLog::Output] Studio Launch Intent is None
12:45:35 RBX [FLog::Output] Is Studio Configured User Id Present: false
12:45:35 RBX [FLog::StudioQtCategoryFilterLogging] No Logging CategoryFilter configuration found for: qt.qpa.input.tablet
12:45:35 RBX [FLog::StudioQtCategoryFilterLogging] No Telemetry CategoryFilter configuration found for: qt.qpa.input.tablet
12:45:36 RBX [FLog::Error] Redundant Flag ID: AcousticSimulation
12:45:36 RBX [FLog::Error] Redundant Flag ID: AdaptiveAnimations
12:45:36 RBX [FLog::Error] Redundant Flag ID: AdvancedCommandBar
12:45:36 RBX [FLog::Error] Redundant Flag ID: AssistantMeshGen
12:45:36 RBX [FLog::Error] Redundant Flag ID: AvatarJointUpgrade
12:45:36 RBX [FLog::Error] Redundant Flag ID: AvatarMakeup
12:45:36 RBX [FLog::Error] Redundant Flag ID: CreateAnimationFromVideo
12:45:36 RBX [FLog::Error] Redundant Flag ID: CreateAssetAsync
12:45:36 RBX [FLog::Error] Redundant Flag ID: CustomVideoUpload
12:45:36 RBX [FLog::Error] Redundant Flag ID: DefaultInstances2
12:45:36 RBX [FLog::Error] Redundant Flag ID: DraggerImprovements2
12:45:36 RBX [FLog::Error] Redundant Flag ID: EmissiveMaps2
12:45:36 RBX [FLog::Error] Redundant Flag ID: FacialAnimationRecording
12:45:36 RBX [FLog::Error] Redundant Flag ID: GltfExport
12:45:36 RBX [FLog::Error] Redundant Flag ID: IEAvatarAutoSetup
12:45:36 RBX [FLog::Error] Redundant Flag ID: LuaConstraintTool
12:45:36 RBX [FLog::Error] Redundant Flag ID: LuauDCR
12:45:36 RBX [FLog::Error] Redundant Flag ID: LuauNonStrictByDefault
12:45:36 RBX [FLog::Error] Redundant Flag ID: NewCameraControls
12:45:36 RBX [FLog::Error] Redundant Flag ID: PartyEmulator
12:45:36 RBX [FLog::Error] Redundant Flag ID: ProximityPromptIndicators
12:45:36 RBX [FLog::Error] Redundant Flag ID: Reimport
12:45:36 RBX [FLog::Error] Redundant Flag ID: RevampedAssetManagerRelease
12:45:36 RBX [FLog::Error] Redundant Flag ID: ScriptSyncM2_5
12:45:36 RBX [FLog::Error] Redundant Flag ID: ServerAuthorityCoreAPI
12:45:36 RBX [FLog::Error] Redundant Flag ID: Texture4k
12:45:36 RBX [FLog::Error] Redundant Flag ID: TextureGenerator2
12:45:36 RBX [FLog::Error] Redundant Flag ID: VideoApi
12:45:36 RBX [FLog::UpdateManager] Update is not enabled due to 'Installation location not supported'
12:45:36 RBX [StudioQtLogging] <empty>: EnableNonClientDpiScaling() failed for HWND 0x10084 (120) (Call not implemented.)
12:45:36 RBX [FLog::WindowGeometryManager] Failed to restore window geometry
12:45:36 RBX [FLog::CreateGraphicsEngine] Potential graphics modes:
12:45:36 RBX [FLog::Graphics] RenderView created[1]
12:45:36 RBX [FLog::Graphics] Loaded 2229 shaders from pack vulkan_desktop variant default (14729219 bytes)
12:45:36 RBX [FLog::Graphics] Compiled 701 shaders in 6 ms
12:45:36 RBX [FLog::Graphics] TM2: newTranscoder:true
12:45:36 RBX [FLog::Graphics] TM2: retry:false, handle406:true, respectDoNotAllowBaseline:true
12:45:36 RBX [FLog::Graphics] TM2: MaxCloudAssetDimension: 1024
12:45:36 RBX [FLog::Graphics] LightGrid is unified
12:45:36 RBX [FLog::Output] Hello world!!!
12:45:36 RBX [DFLog::BootcampCLI149079Log] Hello world?
12:45:36 RBX [DFLog::BootcampCLI154937Log] I love roblox. Roblox is life
12:45:36 RBX [DFLog::BootcampCLI156010Log] Hello there, world
12:45:36 RBX [DFLog::BootcampCLI163626Log] Hello world :)
12:45:36 RBX [DFLog::BootcampCLI164711Log] Hello World :D
12:45:36 RBX [DFLog::BootcampCLI165710Log] Hi everyone!
12:45:36 RBX [DFLog::BootcampCLI179797Log] Hello world from the future!
12:45:36 RBX [DFLog::BootcampCLI179797Log] Another hello world
12:45:36 RBX [StudioQtLogging] <empty>: EnableNonClientDpiScaling() failed for HWND 0x20076 (120) (Call not implemented.)
12:45:36 RBX [FLog::StudioEmbeddedBrowserWebView2] clearAllCookiesAndRunCallback calling CreateCoreWebView2EnvironmentWithOptions
12:45:36 RBX [FLog::StudioEmbeddedBrowserWebView2] CreateCoreWebView2EnvironmentWithOptions: result=0
12:45:36 RBX [FLog::StudioEmbeddedBrowserWebView2] createCoreWebView2ControllerForUser calling CreateCoreWebView2Controller
12:45:36 WIN 03c4:err:winediag:wined3d_dll_init Using the Vulkan renderer.
12:45:36 RBX [FLog::StudioPluginRelay] PluginDockWidgetRelay::initializeFromInfo: rplg_sabuiltin_AudioCompressorEditor.rbxm_AudioCompressorEditor - Failed Empty
12:45:37 RBX [FLog::StudioPluginRelay] PluginWidgetRelay::updateWidgetTitle: rplg_sabuiltin_AudioCompressorEditor.rbxm_AudioCompressorEditor - Failed Empty
12:45:37 WIN 03c4:err:winediag:wined3d_dll_init Using the Vulkan renderer.
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] CreateCoreWebView2Controller: result=0
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] clearAllCookiesAndRunCallbackHelper calling GetCookies
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_FOGENABLE) (0x1c) should have a representative.
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_SPECULARENABLE) (0x1d) should have a representative.
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_COLORKEYENABLE) (0x29) should have a representative.
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_RANGEFOGENABLE) (0x30) should have a representative.
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_LIGHTING) (0x89) should have a representative.
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_COLORVERTEX) (0x8d) should have a representative.
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_LOCALVIEWER) (0x8e) should have a representative.
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_NORMALIZENORMALS) (0x8f) should have a representative.
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_DIFFUSEMATERIALSOURCE) (0x91) should have a representative.
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_SPECULARMATERIALSOURCE) (0x92) should have a representative.
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_AMBIENTMATERIALSOURCE) (0x93) should have a representative.
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_EMISSIVEMATERIALSOURCE) (0x94) should have a representative.
12:45:37 WIN 03c4:err:d3d:validate_state_table State STATE_RENDER(WINED3D_RS_VERTEXBLEND) (0x97) should have a representative.
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] GetCookies: result=0
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] loadUrl calling CreateCoreWebView2EnvironmentWithOptions
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] CreateCoreWebView2EnvironmentWithOptions: result=0
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] CreateCoreWebView2Controller: result=0
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] onCreateCoreWebView2ControllerCompleted calling clearAllCookiesAndRunCallbackHelper
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] clearAllCookiesAndRunCallbackHelper calling GetCookies
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] GetCookies: result=0
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] clearAllCookiesAndRunCallbackHelper: result=0
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] setInitScript calling AddScriptToExecuteOnDocumentCreated
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] Event: CoreWebView2ProcessInfosChanged
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] calling GetProcessExtendedInfos
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] Event: CoreWebView2NavigationStarting
12:45:37 RBX [FLog::StudioEmbeddedBrowserWebView2] GetProcessExtendedInfos: result=0
12:45:38 RBX [FLog::StudioEmbeddedBrowserWebView2] Event: CoreWebView2NavigationStarting
12:45:38 RBX [FLog::StudioEmbeddedBrowserWebView2] AddScriptToExecuteOnDocumentCreated: result=0
12:45:38 RBX [FLog::StudioEmbeddedBrowserWebView2] Event: CoreWebView2ProcessInfosChanged
12:45:38 RBX [FLog::StudioEmbeddedBrowserWebView2] calling GetProcessExtendedInfos
12:45:38 RBX [FLog::StudioEmbeddedBrowserWebView2] GetProcessExtendedInfos: result=0
12:45:39 RBX [FLog::StudioEmbeddedBrowserWebView2] Event: CoreWebView2ProcessInfosChanged
12:45:39 RBX [FLog::StudioEmbeddedBrowserWebView2] calling GetProcessExtendedInfos
12:45:39 RBX [FLog::StudioEmbeddedBrowserWebView2] GetProcessExtendedInfos: result=0
12:45:39 RBX [FLog::StudioEmbeddedBrowserWebView2] Event: DOMContentLoaded
12:45:40 RBX [FLog::StudioEmbeddedBrowserWebView2] Event: CoreWebView2NavigationCompleted
12:45:55 WIN 0380:err:ole:create_server class {daa52b27-8897-50af-ada5-c6c71bb64e17} not registered
12:45:55 WIN 0380:err:ole:com_get_class_object no class object {daa52b27-8897-50af-ada5-c6c71bb64e17} could be created for context 0x4
12:45:55 WIN 0380:err:ole:create_server class {daa52b27-8897-50af-ada5-c6c71bb64e17} not registered
12:45:55 WIN 0380:err:ole:com_get_class_object no class object {daa52b27-8897-50af-ada5-c6c71bb64e17} could be created for context 0x4
12:45:55 WIN 0380:err:ole:create_server class {daa52b27-8897-50af-ada5-c6c71bb64e17} not registered
12:45:55 WIN 0380:err:ole:com_get_class_object no class object {daa52b27-8897-50af-ada5-c6c71bb64e17} could be created for context 0x4
12:46:10 RBX [FLog::StudioEmbeddedBrowserWebView2] Event: CoreWebView2NavigationStarting
12:46:10 RBX [FLog::StudioEmbeddedBrowserWebView2] Event: CoreWebView2NavigationStarting
12:46:10 RBX [FLog::StudioEmbeddedBrowserWebView2] Closing WebView2 controller (EmbeddedAuthorizationDialog), isVisible=false, isLowMemoryMode=false
12:46:10 RBX [FLog::StudioEmbeddedBrowserWebView2] WebView2 controller (EmbeddedAuthorizationDialog) closed
12:46:10 RBX [FLog::StudioEmbeddedBrowserWebView2] WebView2 controller (EmbeddedAuthorizationDialog) closed normally, not force terminating
12:46:11 RBX [FLog::QtUIThread] Not running task `LegacyRunInUIThreadWithQtContext-`: scheduled after UI thread event loop has stopped
12:46:11 RBX [FLog::QtUIThread] Not running task `LegacyRunInUIThreadWithQtContext-`: scheduled after UI thread event loop has stopped
12:46:11 RBX [FLog::QtUIThread] Not running task `LegacyRunInUIThreadWithQtContext-`: scheduled after UI thread event loop has stopped
12:46:11 RBX [FLog::QtUIThread] Not running task `LegacyRunInUIThreadWithQtContext-`: scheduled after UI thread event loop has stopped
12:46:11 RBX [FLog::StudioPluginRelay] PluginDockWidgetRelay::cleanup: rplg_sabuiltin_AudioCompressorEditor.rbxm_AudioCompressorEditor - Failed Empty
12:46:11 RBX [FLog::StudioPluginRelay] PluginDockWidgetRelay::cleanupQWidget: rplg_sabuiltin_AudioCompressorEditor.rbxm_AudioCompressorEditor - Failed Empty
12:46:11 RBX [FLog::CloseDataModel] Clearing World -- 0 Points, 0 Constraints
12:46:11 RBX [FLog::Graphics] RenderView destroyed[1]
12:46:11 RBX [FLog::QtUIThread] Not running task `LegacyRunInUIThreadWithQtContext-`: scheduled after UI thread event loop has stopped
12:46:11 RBX [FLog::QtUIThread] Not running task `LegacyRunInUIThreadWithQtContext-`: scheduled after UI thread event loop has stopped
12:46:11 RBX [FLog::QtUIThread] Not running task `LegacyRunInUIThreadWithQtContext-`: scheduled after UI thread event loop has stopped
12:46:11 RBX [FLog::QtUIThread] Not running task `LegacyRunInUIThreadWithQtContext-`: scheduled after UI thread event loop has stopped
12:46:11 RBX [FLog::AppMemUsageStatus] 1044687361
12:46:15 INF Checking Wineprefix
12:46:15 INF Acquiring user authentication
12:46:15 INF Using user for authentication user=1844782254
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x82576d]

goroutine 355 [running]:
github.com/sewnie/wine.(*RegistryKey).GetValue(...)
	/run/build/vinegar/vendor/github.com/sewnie/wine/registry_key.go:58
main.(*app).getSecurity(0xc0007efb80)
	/run/build/vinegar/cmd/vinegar/app_roblosec.go:40 +0x50d
main.(*bootstrapper).setup(0xc0007100c0)
	/run/build/vinegar/cmd/vinegar/bootstrapper_setup.go:36 +0xcc
main.(*bootstrapper).run(0xc0007100c0, {0x0, 0x0, 0x0})
	/run/build/vinegar/cmd/vinegar/bootstrapper.go:100 +0x136
main.(*manager).run(0xc000b85128)
	/run/build/vinegar/cmd/vinegar/manager_actions.go:39 +0x165
main.(*app).errThread.func1()
	/run/build/vinegar/cmd/vinegar/app.go:193 +0x27
created by main.(*app).errThread in goroutine 1
	/run/build/vinegar/cmd/vinegar/app.go:192 +0x67
```

Rerun:
```bash
$ flatpak run org.vinegarhq.Vinegar
12:46:53 INF Logging to file path=/home/khtsly/.var/app/org.vinegarhq.Vinegar/cache/vinegar/logs/2026-01-06T12:46:52+07:00.log
12:46:53 INF Checking Wineprefix
12:46:53 INF Acquiring user authentication
12:46:53 INF Using user for authentication user=1844782254
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x82576d]

goroutine 50 [running]:
github.com/sewnie/wine.(*RegistryKey).GetValue(...)
	/run/build/vinegar/vendor/github.com/sewnie/wine/registry_key.go:58
main.(*app).getSecurity(0xc00093d000)
	/run/build/vinegar/cmd/vinegar/app_roblosec.go:40 +0x50d
main.(*bootstrapper).setup(0xc00081e0c0)
	/run/build/vinegar/cmd/vinegar/bootstrapper_setup.go:36 +0xcc
main.(*bootstrapper).run(0xc00081e0c0, {0xc000986550, 0x0, 0x0})
	/run/build/vinegar/cmd/vinegar/bootstrapper.go:100 +0x136
main.(*app).commandLine.func1()
	/run/build/vinegar/cmd/vinegar/app.go:153 +0x65
main.(*app).errThread.func1()
	/run/build/vinegar/cmd/vinegar/app.go:193 +0x27
created by main.(*app).errThread in goroutine 1
	/run/build/vinegar/cmd/vinegar/app.go:192 +0x67
```

My Settings:

<img width="606" height="973" alt="Image" src="https://github.com/user-attachments/assets/6a33a77c-6a13-4709-9ac0-30dcc0d6ae87" />

--- a/cmd/vinegar/app_roblosec.go
+++ b/cmd/vinegar/app_roblosec.go
@@ -34,8 +34,6 @@ func (a *app) getSecurity() error {
 	user := keyStream(c, uk.GetValue("Password").Data.([]byte))
 	slog.Info("Using user for authentication", "user", user)
 
-	// Surely Roblox Studio would set the userid and then forget
-	// to create the ROBLOSECURITY cookie for that user?
 	sec := cred.Query(authNamePrefix + `.ROBLOSECURITY` + user)
 	if sec == nil {
 	    slog.Warn("ROBLOSECURITY cookie not found", "user", user)

